### DISCUSSION TITLE
Textual Console Dev

### DISCUSSION THREAD

**Discussion by ThomasJRyan** [darrenburns/posting#157] [Q&A] [answered]
I feel silly for even asking this, but I've been wondering how people go about invoking the textual console when working with Posting's codebase.

My immediate thought would be to run `uv run textual run --dev src/posting/__main__.py` but that is seemingly incorrect

My next thought would be to create a new file which runs the app and go from there but it feels like there must be a better option

**Comment by darrenburns** [comment, marked as answer]
You can run 'TEXTUAL=debug,devtools posting' 🙂 not silly at all!

**Reply by ThomasJRyan** [reply]
Thank you very much Darren 🙏
